utils.py

```diff
@@ -420,6 +437,13 @@ def get_human_readable_time_string(time_msec):
 def are_datetimes_close(later_datetime, earlier_datetime):
     """Given two datetimes, determines whether they are separated by less than
     feconf.PROXIMAL_TIMEDELTA_SECS seconds.
+    
+    Arg:
+        later_datetime: function. Later date and time.
```

Comment:
I think the arg type is incorrect, can you please check here and elsewhere.

Reply:
done